app/controllers/experiences_controller.rb

```diff
@@ -19,5 +20,13 @@ def show
   end
 
   def new
+    @experience = Experience.new
+    @experience.user = current_user
+    @experience.status = "building"
```

Comment:
if you have `default: 'building'` in your migration then you wouldn't need this line